tree_test.go

```diff
@@ -1232,7 +1190,7 @@ func genRandomKeyValues(rand *mRand.Rand, count int) []keyValue {
 	return ret
 }
 
-func BenchmarkBatchLeavesInsert(b *testing.B) {
+func BenchmarkNewLeavesInsert(b *testing.B) {
```

Comment:
Now this benchmark uses the base API since should be the fastest way to insert new leaves; so got simpler.